### PR TITLE
#Prepare project for TSE review 

### PR DESCRIPTION
**Pull request by codecell**
**What does this PR do?**
- gets the code base ready for TSE review

**How can this manually Tested**
-once this repo is cloned
-cd into the project
-in your terminal run `ruby bubble_sort.rb`
-in your terminal, you should see the result of all the arrays passed into the **bubble_sort** and **bubble_sort_by** methods sorted in **ascending** order.

### CODE REVIEW

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  arr.each do
+    (size - 1).times do |i|
+      arr[i], arr[i + 1] = arr[i + 1], arr[i] if arr[i] > arr[i + 1]
+    end
+  end
+
+  print arr.to_a
+  puts
+end
```
Comment (stratospherique) [inline bubble_sort.rb]: - The method should **return** the sorted array and not **print** it. :warning:

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  arr.each do
+    (size - 1).times do |i|
+      arr[i], arr[i + 1] = arr[i + 1], arr[i] if yield(arr[i], arr[i + 1]).positive?
+    end
+  end
+
+  print arr.to_a
+  puts
+end
```
Comment (stratospherique) [inline bubble_sort.rb]: - The method should **return** the sorted array and not **print** it. :mag:

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great start @team :clap: However, the methods need further refactoring in order for us to approve the project. Check :arrow_down: 

- **[REQUIRED]** Check the code comments below, they should help you refactor your method :gear: 

### That's it :relieved: When finished, kindly resubmit another code review request :pray: 

Happy coding :v: 
**_Ahmed Mahfoudh_**

**Review by Forison — APPROVED** [review]
## Status::Approved💚
# 🙋‍♂️Njoroge & Alfred 
Good job on implementing the changes raised in the previous review. BOTH methods behave as required.
Go ahead and merge branch💪 
#### Happy coding 💻
#### Addo Forison

**Pull request merged**